contrib/debian/copyright

```diff
@@ -1398,51 +1394,3 @@ License: Apache-Qpid-Proton-with-BSD-Subcomponents
    See the License for the specific language governing permissions and
    limitations under the License.
  .
- .
- PROTON SUBCOMPONENTS:
- .
- Proton includes freegetopt with a separate BSD license.  Your use
- of the source code for freegetopt is subject to the terms and
- conditions of its license in examples/include/pncompat/internal/LICENSE.
- .
- The setup scripts for the python bindings include files derived by
- PyZMQ and are licensed with a separate Modified BSD license.  Use of
- the source code in these setup files are subject to the terms and
- conditions in the license:
- proton-c/bindings/python/setuputils/PYZMQ_LICENSE.BSD.
-
-License: GNU-All-permissive-License
```

Comment:
This and `License: GPLv3-or-later-with-Autoconf-exception` are different licenses that should be retained.

Reply:
This hasn't been resolved.